test/index.coffee

```diff
@@ -6,14 +6,16 @@ proxyquire = require 'proxyquire'
 gutilStub =
   log: ->
     #console.log(arguments)
+fancyLogStub = ->
+  gutilStub.log.apply(null, arguments)
```

Comment:
as tests are expecting to be able to spy on an object method, I used this trick to spy on `fancy-log`, which is a function directly. I welcome any better solution but I have no experience with sinon.